src/page-multi-facility/SystemTypeSelection.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import styled from "styled-components";
+
+import InputSelect from "../design-system/InputSelect";
+
+const SystemTypeInputDiv = styled.div`
+  width: 195px;
+  padding-bottom: 20px;
+`;
+
+interface Props {
+  systemType?: string;
+  setSystemType: (systemType?: string) => void;
+}
+
+const SystemTypeSelection: React.FC<Props> = ({
+  systemType,
+  setSystemType,
+}) => {
+  const systemTypeList = [
+    { value: undefined },
+    { value: "State Prison" },
+    { value: "County Jail" },
```

Comment:
considering how important these values are now I feel like they should be in a shared constant somewhere instead of just being magic strings? I'm not sure where else (if anywhere) we really refer to them explicitly at this point but still